internal/fsstat/dev.go

```diff
@@ -0,0 +1,23 @@
+// Package fsstat provides access to platform specific file stat info
```

Comment:
Trailing dot `.` to complete the sentence, please.
Here and in other doc comments.
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
```suggestion
// Package fsstat provides access to platform specific file stat info.
```